modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/provider/zoom/ZoomPowwowServiceProvider.java

```diff
@@ -185,46 +185,42 @@ public boolean isSupportsOptionPassword() {
 		User user, PowwowServer powwowServer, long powwowMeetingId, String name,
 		Map<String, String> options) {
 
-		Map<String, String> parameterMap = new HashMap<>();
-
 		String hostId = getHostId(user, powwowServer);
 
-		parameterMap.put("host_id", hostId);
-
-		parameterMap.put(
-			"option_host_video",
-			options.get(PowwowMeetingConstants.OPTION_AUTO_START_VIDEO));
-		parameterMap.put(
-			"option_participants_video",
-			options.get(PowwowMeetingConstants.OPTION_AUTO_START_VIDEO));
+		List<String> resourceParams = new ArrayList<>(3);
+		resourceParams.add("users");
+		resourceParams.add(hostId);
+		resourceParams.add("meetings");
 
 		String password = options.get(PowwowMeetingConstants.OPTION_PASSWORD);
 
+		// Create Meeting JSONObject
```

Comment:
instead of this comment, should we collect to a private method say `createMeetingJSON`?

Reply:
updated